README.md

```diff
@@ -365,3 +392,85 @@ $ curl https://localhost:4430 -k -v
 ```
 
 Notice that the issue in this example is being signed by a training Venafi TPP instance which is not a trusted certificate authority.
+
+### Securing an Ingress
+
+cert-manager can also be used to secure incoming traffic to your cluster.
+To demonstrate a working ingress we built a sample "Hello World" service in `hello-world.yaml`.
+In this file we have an Ingress entry. Ingresses can be automatically secured by cert-mananger using special annotations on the Ingress resource.
+
+![](./images/NGINXPlus-K8s.svg)
+
+This is a diagram of what we're building in this part of the lab.
+
+Open the `hello-world.yaml` file and change the host fields in the file to match the external hostname of your instance so we can test this from the outside world.
+This can be found in your CloudShare environment under "Connection Details", then "External Address".
+
+The `cert-manager.io/issuer` annotation tells cert-manager to install a TLS certificate received from the Issuer we installed earlier.
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: hello-world-ingress
+  namespace: default
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    cert-manager.io/issuer: "venafi-tpp-issuer"
+spec:
+  tls:
+    - hosts:
+        - <place external hostname here>
+      secretName: venafi-demo-tls
+  rules:
+    - host: <place external hostname here>
+    [...]
+```
+
+After changing and saving the file we can deploy it using `kubectl apply -f hello-world.yaml`
+
+Now we have an Ingress deployed with a `cert-manager.io/issuer` annotation, this will tell cert-manager to use a specific issuer to automatically fetch and assign a certificate according to the policy you have configured on your Issuer resource.
+We can see them being issued using:
+```console
+$ kubectl get certificate
+NAME              READY   SECRET            AGE
+venafi-demo-tls   True    venafi-demo-tls   1m
+```
+The private key and certificate have been saved into `venafi-demo-tls` inside the Kubernetes secret store from where NGINX picks it up.
+The NGINX Ingress controller has native support for reading TLS secrets from Kubernetes.
+
+### Testing the deployment
+
+#### Using curl
+You can see it being served using:
+```console
+$ curl https://<hostname> -k -v
```

Comment:
It seems a bit weird to have the flags after the URL. Consider putting the URL last.